### PR TITLE
HTML Template created

### PR DESCRIPTION
**Pull request by namanjaidka02**

### CODE REVIEW

File: Task 4/google-search-engine-clone.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
+body{
```
Comment (jaidkasahil) [inline Task 4/google-search-engine-clone.css:1]: Change file name to index.css

File: Task 4/google-search-engine-clone.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```
Comment (jaidkasahil) [inline Task 4/google-search-engine-clone.html:1]: Change file name to index.html

Comment (jaidkasahil) [reply Task 4/google-search-engine-clone.html:1]: if only 1 html file name it index.html

File: Task 4/google-search-engine-clone.css (hunk 1 above)
Comment (jaidkasahil) [reply Task 4/google-search-engine-clone.css:1]: Delete empty files

File: Task 4/google-search-engine-clone.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    margin: 0;
+    font-family: Arial, sans-serif;
+    font-size: 0.9em;
+    max-width: 100%;
```
Comment (jaidkasahil) [inline Task 4/google-search-engine-clone.css:5]: Why this???

File: Task 4/google-search-engine-clone.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    <title>Google</title>
+    <link
+      rel="icon"
+      href="https://th.bing.com/th/id/OIP.lp6AjI9vfG4iLCicENccwQHaHa?w=150&h=150&c=7&r=0&o=5&dpr=1.25&pid=1.7"
+    />
+    <link rel="stylesheet" href="/google-search-engine-clone.css" />
+    <script
+      src="https://kit.fontawesome.com/992348ea18.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (jaidkasahil) [inline Task 4/google-search-engine-clone.html:16]: Explain this

Comment (namanjaidka02) [reply Task 4/google-search-engine-clone.html:16]: emported mic menu bar and and search icon from this website.

File: Task 4/google-search-engine-clone.html (hunk 2 above)
Comment (namanjaidka02) [reply Task 4/google-search-engine-clone.html:1]: okay.

**Pull request merged**